### PR TITLE
Add hseval

### PR DESCRIPTION
**Pull request by moonheart08**
Added a haskell sandbox
https://hackage.haskell.org/package/mueval
use the manual download setup, not the cabal setup, or it wont work right! (last i checked)
EDIT: Wow, banned after i made this thing. someone mind telling me why?

### CODE REVIEW

**Comment by rgoliveira** [comment]
@moonheart08 I wass afk for a while. Someone else got the bot token and was messing around with it, and let you take the fall. You're unbanned and more than welcome to come back. Me and the rest of the Mods team apologize for the misunderstanding.

**Comment by moonheart08** [comment]
Oh, that explains it, Nice to know, thanks! I hope you regain control of the bot :p

File: commands/hseval/hseval.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    out += d.toString();
+  });
+  mueval.on('exit', () => {
+    cb(out);
+  });
+}
+module.exports = {
+  usage: 'hseval <haskell expression> - runs <haskell expression> using mueval as the sandbox',
+  run: (bot, message, args) => {
+    if (!args.length) return true;
```
Comment (Samoxive) [inline commands/hseval/hseval.js]: you could use `!args` instead in if block, !'' is truthy in javascript.

Comment (moonheart08) [inline commands/hseval/hseval.js]: Oh, true! That is correct. I was just being explict in my code because thats how i was thinking at the time

**Comment by rgoliveira** [comment]
Ok, we won't be merging this. I'm already working integrating the bot with repl.it.

Thanks for the contribution, though! :+1:

**Pull request closed without merging**